tests/plugins-tester/.wp-env.json

```diff
@@ -1,16 +1,17 @@
 {
-    "phpVersion": "7.4",
-    "plugins": [
-	  "./plugins/elementor-tests-utils",
-	  "../../"
-    ],
-    "themes": [
-        "https://downloads.wordpress.org/theme/hello-elementor.zip"
-    ],
-    "mappings": {
-        "elementor-templates": "./elementor-templates",
-        "scripts": "./scripts"
-    },
-    "port": 7777,
-    "testsPort": 7778
+	"phpVersion": "7.4",
```

Comment:
7.4 isn't supported [anymore](https://www.php.net/supported-versions.php). Do we consider switching to more fresh version?

Reply:
We use the same for playwright. I'm not the expert here.

Reply:
@mykytamurzin 
@dmaziar 
we support 7.0+ versions

Reply:
version 8.2 causes a lot of PHP Deprecated warnings. If we consider changing the version, it should be in a separate PR